apps/consulting/posts/openai-pseudocode.md

```diff
@@ -0,0 +1,274 @@
+---
+title: 'Exploring the Impact of Feedback Loops on LLM Code Generation'
+published: March 27, 2023
+author: chris-ostrouchov
+description: 'As an experiment we wanted to add a feedback loop into LLM generated code via type annotations, docstrings, automated tests, and user feedback. We developed a simple package pseudoscode which uses the OpenAI API for generating code.'
+category: ["Artificial Intelligence"]
+featuredImage:
+  src: '/posts/openai-pseudocode/pseudocode-screenshot.png'
+  alt: 'A screenshot of the pseudocode program running'
+hero:
+  imageSrc: /posts/hero-paris.webp
+  imageAlt: 'Data visualization of Paris city'
+---
+
+<base target="_blank" />
+
+## Overview
+
+Large Language Models (LLMs) have been around for several years but
+recent advances have revolutionized the field of natural language
+processing (NLP) and artificial intelligence (AI), opening up a world
+of possibilities across various domains. [OpenAI's
+ChatGPT](https://chat.openai.com/) has taken the world by storm and
+shown remarkable ability to generate human-like responses to a wide
+range of queries.
+
+These models require training on vast amounts of data and in some
+sense can be thought of as a way to summarize and expose the
+information in the training dataset. It is important to remember
+however that the information returned in response to a query is a
+statistical answer and these models have no concept of
+'correctness'. This leads to the so called hallucination effect that
```

Comment:
```suggestion
'correctness.' This leads to the so called hallucination effect that
```

Reply:
We have a suggested new intro in the comments, which would replace this sentence.
https://github.com/Quansight/Quansight-website/pull/703#issuecomment-1494634773